src/lib/openapi/meta-schema-rules.test.ts

```diff
@@ -79,6 +79,7 @@ const metaRules: Rule[] = [
             },
         },
         knownExceptions: [
+            'adminCountSchema',
```

Comment:
I think we should avoid adding more exceptions to the meta-schema-rules and instead add the descriptions

Reply:
Fair point. Addressed in https://github.com/Unleash/unleash/pull/3829/commits/4c52804bb3ec68d675edcf884ff041093e4e3b52